packages/edit-site/src/components/sidebar/global-styles-v2/components/screen-header.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	Heading,
+	HStack,
```

Comment:
We'll need to audit all those imports for missing AND experimental ones. Some components already exist, some exist but are experimental, some don't exist at all.